crosswalks/MODS.csv

```diff
@@ -0,0 +1,68 @@
+﻿Property,MODS
+codeRepository,relatedItem/[@type='Host']
+programmingLanguage,
+runtimePlatform,
+targetProduct,
+applicationCategory,
+applicationSubCategory,
+downloadUrl,location/url[@displayName='Download URL']
+fileSize,physicalDescription/extent
+installUrl,location/url[@displayName='Install URL']
+memoryRequirements,
+operatingSystem,
+permissions,
+processorRequirements,
+releaseNotes,description

```

Comment:
what's the rationale for this? Do you have a link to the spec for `<description>`? I don't see it on https://www.loc.gov/standards/mods/userguide/generalapp.html

Reply:
```suggestion
releaseNotes,

```